ldapauthenticator/ldapauthenticator.py

```diff
@@ -397,7 +397,10 @@ def get_user_attributes(self, conn, userdn):
         attrs = {}
         if self.auth_state_attributes:
             found = conn.search(
-                userdn, "(objectClass=*)", attributes=self.auth_state_attributes
+                search_base=userdn,
```

Comment:
Understanding that this param was `search_base` helped me understand things better.